specification/ai/Azure.AI.Projects/connections/models.tsp

```diff
@@ -56,3 +59,23 @@ union ConnectionType {
   @doc("Custom Keys")
   Custom: "CustomKeys",
 }
+
+@doc("The authentication type used by the connection")
+union AuthenticationType {
+  string,
+
+  @doc("API Key authentication")
```

Comment:
Due to Microsoft's SFI initiative, Azure services are supposed to support only Entra ID now and NOT support api-key or SAS (unless it is delegated SAS). If you need to support api-key or non-delegated SAS for getting-started scenarios (but not for customer long-term production use scenarios), then a short expiration (perhaps a week or so) on the key/sas is required.

Reply:
@JeffreyRichter thanks for the note, I'll follow up with you offline on this.